### DISCUSSION TITLE
Help for bypass and click

### DISCUSSION THREAD

**Discussion by GkyEla** [FlareSolverr/FlareSolverr#1168] [Q&A]
Hello I read all the document but ididnt understand where should i edit or write my python script at i am really confused sorry. Windows x64 executable. Ijust need to bypass cloudflare than click to a button at site and than refresh with new proxy and again

**Comment by ilike2burnthing** [comment]
If you're on Windows, you can use https://github.com/ilike2burnthing/FlareSolverr/tree/looping and follow the instructions in the readme to run from source. I wasn't able to get it working with proxies or a VPN, but perhaps you will have better luck than me.
